bin/GameIndex.dbf

```diff
@@ -40213,6 +40242,12 @@ Name   = Metal Gear Solid 3 - Subsistence [Disc1of3]
 Region = NTSC-U
 Compat = 5
 ---------------------------------------------
+Serial = SLUS-21243
```

Comment:
Ah I didn't know that , I thought they were alphabetically , well at least some.
ocd gets in the way xD